docker/Makefile

```diff
@@ -39,9 +36,8 @@ ifeq ($(IS_LATEST),true)
 endif
 
 test: prep
-	@bundle install --path $$BUNDLE_PATH --gemfile $$GEMFILE
```

Comment:
Will the jenkins worker be explicitly `bundle install`ing in the `docker` directory?

For local dev I find this useful since I can just run like `make build test` and not need to worry about anything else, but this is up to you / your team.

Reply:
I arbitrarily chose to do that in jenkins shell script https://github.com/puppetlabs/ci-job-configs/pull/6153/files#diff-600fe5e48a6aa68a129677e6871db8deR46 If we go with allowing these to be overriden by environment variables I think it would make sense to keep it.